src/components/Dropdown/Dropdown.js

```diff
@@ -0,0 +1,69 @@
+import * as React from 'react';
+import List from '@mui/material/List';
+import ListItem from '@mui/material/ListItem';
+import MenuItem from '@mui/material/MenuItem';
+import Menu from '@mui/material/Menu';
+import SystemTypography from '../Typography/Typography';
+import KeyboardArrowDownIcon from '@mui/icons-material/KeyboardArrowDown';
+
+export default function AppDropdown(props) {
+  const {optionsArray, dropdownTitle} = props;
+  const options = optionsArray;
+
+  const [anchorEl, setAnchorEl] = React.useState(null);
```

Comment:
AppDropdown state must be dynamic, this anchorEl state will not be able to reach out our spiels